ShivamShukla.java

```diff
@@ -0,0 +1,59 @@
+import java.io.*;
+
+import java.util.*;
+
+import java.util.regex.*;
+
+class Shivam
+{ 
+public static void main(String args[])
+
+{ Scanner sc=new Scanner(System.in);
+  
+int n;
+
+String text=sc.nextLine();
+
+n=Integer.parseInt(sc.nextLine());
+
+pass(text,n);
+
+}
+
+public static void pass(String t,int n)
+
+{
+String temp=t;
+Scanner sc=new Scanner(System.in);
+int i;
+String a[]=new String[100];
```

Comment:
Avoid static allocation. if n is 5 then the rest of the memory will be wasted. if n will go more than 100 it will throw IndexOutOfBoundException.